src/pages/community/patterns/export-pattern/index.mdx

```diff
@@ -150,3 +151,47 @@ A notification can be used to alert the user that an export has been successful.
 
 </Column>
 </Row>
+
+## Accessibility
+
+Adhere to
+[Carbon's modal guidelines](https://www.carbondesignsystem.com/components/modal/accessibility).
+
+### Trigger
+
+Most often, the export modal is triggered by clicking an "export" button.
+
+1. The user navigates to the Export button using the tab key. If Export is
+   launched from an overflow menu, the user may navigate to Export within the
+   overflow menu using the up/down arrow keys.
+2. The focus state is applied to the button/menu item.
+3. The user clicks "enter" to launch the Export modal.
+
+### Focus
+
+When the modal opens, focus is set depending on whether sensible default choices
+are defined or not.
+
+#### If the user must complete fields before the export can be executed:
+
+1. On opening the modal, the focus is applied to the first field that accepts
+   user input.
+2. On `tab`, progress through any further fields that accept user input.
+3. On `tab`, apply focus to "Export".
+4. On `tab`, apply focus to "Cancel".
+
+#### If a sensible default value is provided:
+
+1. On opening the modal, set the focus to the "Export" button.
+2. On `tab`, apply focus to "Cancel".
+3. On `tab`, apply focus to the first field that accepts user input. Follow
+   accessibility guidance for the interaction type of this field (i.e.
+   [Radio button](https://www.carbondesignsystem.com/components/radio-button/accessibility#how-it-works)
+   or
+   [Text input](https://www.carbondesignsystem.com/components/text-input/accessibility#how-it-works))
+4. On `tab`, progress through any further fields that accept user input.
+
+### Feedback
+
+Where passive modals are used to alert the user of success or failure, open the
+modal with focus applied to the "Close" icon.
```

Comment:
```suggestion
modal with focus applied to the close icon.
```

Reply:
@chloepoulterdesign I'm guessing this sentence wasn't meant to go under the feedback section?

Reply:
I think that feedback section is badly named - it's meant to refer to the user interaction where they recieve feedback on the successful export. Will rename!